client/state/data-layer/wpcom/signup/verticals/index.js

```diff
@@ -31,6 +31,7 @@ export const requestVerticals = action =>
 				...( action.siteTypeId && { site_type: action.siteTypeId } ),
 				limit: action.limit,
 				include_preview: true,
+				synonyms: true,
```

Comment:
Following the suggested update to D32511-code

```suggestion
				allow_synonyms: true,
```